### ISSUE TITLE
Can You Provide A JAR File?

### ISSUE THREAD

**Issue by thirtyxarcher** [tmoran3020/binding-airscape#1]
Any possibility you could provide a JAR file or, even better, walk me through how to create one? What you've got here looks great. Thanks.

**Comment by tmoran3020** [comment]
I just built it using the [dev instructions](https://www.openhab.org/docs/developer/), but to actually build it yourself (without the skeleton script) I think you'd need to change the parent pom. I haven't worked on submitting it formally, this was mostly my way of backing up my work as I went on to another module (it does work though). Let me know what version of Openhab you're on and I can build a jar, though, too.

**Comment by thirtyxarcher** [comment]
Tim,
Thanks for the prompt response. It would be awesome if you'd build a jar
for me. I'm just getting started in OpenHAB and am on version 2.5.6. I
really appreciate your help. Have a great weekend.

**Comment by tmoran3020** [comment]
Checked into /dist, let me know how that works for you. I test it by just adding to a local addons, so to my knowledge it should "just work."

Note that adding an hour is a bit funky - openhab isn't great about buttons. So I created a switch that will do it "addOneHour", but it's clunky to turn it off and on again each time. There are some threads about how to fake a button, but I haven't worried about it yet.
